.vsts-dotnet.yml

```diff
@@ -15,8 +14,15 @@ parameters:
   displayName: Optional OptProfDrop Override
   type: string
   default: 'default'
+  default: true
```

Comment:
Oops, after merging this problem was found. default is defined twice and the encountered the error:/.vsts-dotnet.yml (Line: 17, Col: 3): 'default' is already defined.